esphome/components/nextion/nextion.cpp

```diff
@@ -384,6 +391,12 @@ void Nextion::process_nextion_commands_() {
   this->print_queue_members_();
 #endif
   while ((to_process_length = this->command_data_.find(COMMAND_DELIMITER)) != std::string::npos) {
+#ifdef USE_NEXTION_MAX_COMMANDS_PER_LOOP
+    if (++commands_processed > this->max_commands_per_loop_) {
+      ESP_LOGW(TAG, "Processed too many commands in one loop. Deferring remaining commands.");
```

Comment:
Try to keep messages [as short as possible.](https://developers.esphome.io/architecture/logging/)
```suggestion
      ESP_LOGW(TAG, "Command processing limit exceeded");
```

Reply:
Good point. That will save a few bytes...
I will review other strings on this component (in another PR).